x-pack/legacy/plugins/uptime/server/rest_api/pings/get_all.ts

```diff
@@ -25,19 +25,29 @@ export const createGetAllRoute: UMRestApiRouteCreator = (libs: UMServerLibs) =>
   options: {
     tags: ['access:uptime'],
   },
-  handler: async (_context, request, response): Promise<any> => {
-    const { size, sort, dateRangeStart, dateRangeEnd, location, monitorId, status } = request.query;
+  handler: async (
+    {
+      core: {
+        elasticsearch: {
+          dataClient: { callAsCurrentUser },
```

Comment:
This is quite a bit of boilerplate in a  function signature. Any thoughts to how we could simplify this destructuring in the future?

Reply:
Well we rely on that field. Alternatively you could do something like:

```typescript
const func = (request) => {
  const { callAsCurrentUser } = request.core.elasticsearch.dataClient;
  // ...
}
```

Reply:
@andrewvc please review https://github.com/elastic/kibana/pull/51403/commits/c745fb816e23fb0fd2f38c221905a232210943a8